test/kata4.humanCatDogYears.test.js

```diff
@@ -1,3 +1,13 @@
 const { humanCatDogYears } = require("../src");
 
-// Look Ma, no handlebars!!!
+describe("humanCatDogYears", () => {
+    test("returning array of human, cat and dog years", () => {
```

Comment:
Too much indentation, 2 spaces for line 4, 6, 7 and 9, and 4 spaces for line 5 and 8